xiaogpt/xiaogpt.py

```diff
@@ -94,14 +94,23 @@ async def poll_latest_ask(self):
             session._cookie_jar = self.cookie_jar
             while True:
                 self.log.debug(
-                    "Now listening xiaoai new message timestamp: %s",
-                    self.last_timestamp,
+                    "listening new message, timestamp: %s",
+                    self.last_timestamp
```

Comment:
f-string seems ok here
e.g. f"listening new message, timestamp: {self.last_timestamp}"

Reply:
no, the current form is preferred, it's not a string formatting, its lazy interpolation

Reply:
copy that @huwei901108 forget me...